### ISSUE TITLE
single lidar calibration

### ISSUE THREAD

**Issue by dingjiangang** [tudelft-iv/multi_sensor_calibration#25]
so i have a question, is it work for single lidar and vehicle calibration? this code is ok for single lidar calibration?

**Comment by RonaldEnsing** [comment]
No, this package only calibrates the extrinsics of lidar, radar and/or (stereo) camera. One of the sensors is assumed to be already calibrated to the frame of the vehicle.
